db/warehouse/migrate/20241207185500_create_hmis_workflows.rb

```diff
@@ -28,7 +28,7 @@ def change
       t.string :name
       t.references :swimlane, foreign_key: { to_table: :wfd_swimlanes }
       # task nodes have forms.
-      t.references :form_definition, foreign_key: { to_table: :hmis_form_definitions }
+      t.string :form_definition_identifier
       # gateway nodes have types
       t.string :gateway_type
 
```

Comment:
Changes look good to me, just noting I had to split out "up" and "down" functions in order to properly rerun this migration locally, in case others need to do the same:
```
  def down
    execute 'DROP TABLE IF EXISTS wfe_audit_events CASCADE'
    execute 'DROP TABLE IF EXISTS wfe_step_assignments CASCADE'
    execute 'DROP TABLE IF EXISTS wfe_steps CASCADE'
    execute 'DROP TABLE IF EXISTS wfe_instances CASCADE'
    execute 'DROP TABLE IF EXISTS wfd_flows CASCADE'
    execute 'DROP TABLE IF EXISTS wfd_nodes CASCADE'
    execute 'DROP TABLE IF EXISTS wfd_swimlanes CASCADE'
    execute 'DROP TABLE IF EXISTS wfd_templates CASCADE'
  end
```

Reply:
maybe its worth adding this into the migration so its reversible for everyone